alchemist-physical-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/properties/PhysicalPedestrian.kt

```diff
@@ -27,15 +27,15 @@ import it.unibo.alchemist.model.interfaces.Node.Companion.asPropertyOrNull
 /**
  * Base implementation of a pedestrian's capability to experience physical interactions.
  */
-class PhysicalPedestrian<T, P, A, F>(
+abstract class PhysicalPedestrian<T, P, A, F>(
```

Comment:
Whi is this called "pedestrian"? Isn't this just a node with some physics?

Reply:
It captures elements such as *comfortArea* and *comfortRay* which i though are typical of a pedestrian.
However, there is a more general interface called `PhysicalProperty`.